jruby-gradle-base-plugin/build.gradle

```diff
@@ -39,7 +47,7 @@ dependencies {
         exclude module : 'groovy-all'
     }
 
-    provided "org.jruby:jruby-complete:${jrubyVersion}"
+    compile "org.jruby:jruby-complete:${jrubyVersion}"
```

Comment:
hoops, this should not be there. not sure how this got in. had classloader
problems in the beginning ;)